src/components/Viewer/Viewer.js

```diff
@@ -199,12 +217,17 @@ const Viewer = () => {
                 >
                     <ReplayIcon />
                 </IconButton>
-                <Button
+                <FileUploader
+                    acceptedFiles={[
+                        'image/svg+xml',
+                    ]}
+                    caption='Load SVG'
                     className={classes.buttonLoad}
-                    color='primary'
-                >
-                    Load SVG
-                </Button>
+                    error='An error has occured'
+                    onSave={files => {
```

Comment:
No need to create this function you can simply pass files to saveSVG and let saveSVG do files[0].